jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/DataTablesTests.java

```diff
@@ -238,6 +237,18 @@ public static void rowSelectionTableTest() {
         verifyElements(3, 2);
     }
 
+    @Test
+    public static void headerCheckboxTest() {
+        rowSelectionTable.show();
+        rowSelectionTable.clickOnHeaderCheckbox();
+        rowSelectionTable.has().greyHeaderCheckbox();
+        rowSelectionTable.has().areAllCheckboxesChecked(1,true);
+
+        rowSelectionTable.clickOnHeaderCheckbox();
+        rowSelectionTable.has().areAllCheckboxesChecked(1,false);
+        rowSelectionTable.has().blankHeaderCheckbox();
+    }
```

Comment:
Sorry :) but you do not check color here. You check whether checkbox marked or not. Color is either rgba or material. Please see examples in other tests and create proper methods and tests :)

Reply:
done